pnpm-workspace.yaml

```diff
@@ -1,3 +1,2 @@
 packages:
   - packages/*
-  - docs
```

Comment:
I moved this out for now to avoid running `build` on docs when it doesn't depend on anything in the workspace.

Instead, we may want to treat this as a separate app and adjust our `docs` scripts throughout to generate files into this `docs/`.